public/javascript/main.js

```diff
@@ -377,4 +377,16 @@ $(function(){
           $(this).data('chart', chart);
       }
   })
+
+  // Event tracking
+  // Clicks on elements with data-ga-track-click attribute will be recorded
+  $('[data-ga-track-click]').on('click', function(){ 
+    trackClick( $(this).attr('data-ga-track-click')); 
+
+    function trackClick(eventDescription){
+      eventDescription = eventDescription || 'Undescribed event.';
+      ga('send', 'event', 'user interaction', eventDescription, document.title);
+    }
```

Comment:
Seems odd to define this function *in* the event handler and rely on JavaScript's function hoisting. I think it would be better to move this function out of the event handler.